### PR TITLE
enhance(must-gather): Adds integration test for must-gather

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

This commit adds integration test for must-gather which checks whether the commands output collection functionality of ocs-must-gather.

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -33,14 +33,17 @@ DEFAULT_REGISTRY_NAMESPACE="ocs-dev"
...
+REGISTRY_NAMESPACE="${REGISTRY_NAMESPACE:-${DEFAULT_REGISTRY_NAMESPACE}}"
 IMAGE_NAME="ocs-registry"
 IMAGE_TAG="${IMAGE_TAG:-latest}"
 FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGE_NAME}:${IMAGE_TAG}"
 
 default_local_storage_image="quay.io/gnufied/local-registry:v4.2.0"
 LOCAL_STORAGE_IMAGE_NAME="${LOCAL_STORAGE_IMAGE:-$default_local_storage_image}"
 
+MUST_GATHER_IMAGE_NAME="ocs-must-gather"
+MUST_GATHER_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${MUST_GATHER_IMAGE_NAME}:${IMAGE_TAG}"
```
Comment (kshlm) [inline hack/common.sh]: You'll need to switch to the image built by the CI to test the correct image during the functional test. Without this, you'll end up testing the already public must-gather image in quay.

Like the registry image, an image is built by the CI for each PR. This image will be named, as the registry image name below, as `registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:ocs-must-gather-quay`. Note that the tag for the must gather image has a suffix of `-quay`. Images with tag suffixed `-quay`, are the actual images that end up being pushed to quay. The must-gather image doesn't need any customization to run in Openshift CI (unlike the operator and registry images), so we don't have CI specific build of it.

Comment (ashishranjan738) [reply hack/common.sh]: okay, I have made the changes. @kshlm Can you check if that's the correct image or not?

**Pull request merged**